src/navigator.cpp

```diff
@@ -0,0 +1,58 @@
+#include "navigator.h"
+#include "Graph.h"
+
+// File: navigator.cpp
+// Purpose: Class implementing Dijkstra's Algorithm for finding the shortest path in a graph.
+
+
+class Navigator : Graph {
+    private:
+    public:
+        // Construct Navigator with empty Graph
+        Navigator() {};
+
+	// Destructor
+	~Navigator() {};
+
+	// Returns a list of vertices representing the shortest path from X to Y.
+	std::list<vertex> search (vertex *x, 
```

Comment:
Maybe call arguments `*start` and `*finish`?

I was looking at the Google C++ styling and I wanted to adhere to them as much as possible (I'll update that in the Wiki shortly).  It mentions to avoid ambiguous names when possible.

Reply:
I'll await your post in the Wiki on the coding standard and edit to conform.